MapboxDirections/MBVisualInstructionBanner.swift

```diff
@@ -21,6 +21,11 @@ open class VisualInstructionBanner: NSObject, NSSecureCoding {
      */
     @objc public let secondaryInstruction: VisualInstruction?
     
+    /**
+     A visual instruction that is presented simultaneously to provide information about an additional maneuver that occurs in rapid succession.
+     */
+    @objc public let tertiaryInstruction: VisualInstruction?
```

Comment:
This is good (abstracted because it's _either_ a lane, _or_ a turn component), but it may be nice to specifically call out how this property is used in our current implementation.